bundles/org.openhab.core/src/main/java/org/openhab/core/library/unit/SmartHomeUnits.java

```diff
@@ -122,9 +122,16 @@
     public static final Unit<Energy> WATT_HOUR = addUnit(new ProductUnit<>(Units.WATT.multiply(Units.HOUR)));
     public static final Unit<Energy> KILOWATT_HOUR = addUnit(MetricPrefix.KILO(WATT_HOUR));
     public static final Unit<Energy> MEGAWATT_HOUR = addUnit(MetricPrefix.MEGA(WATT_HOUR));
-    public static final Unit<Power> KILOVAR = addUnit(MetricPrefix.KILO(new BaseUnit<Power>("var")));
-    public static final Unit<Energy> KILOVAR_HOUR = addUnit(new ProductUnit<>(KILOVAR.divide(Units.HOUR)),
+    public static final Unit<Power> VAR = addUnit(new AlternateUnit<Power>(Units.WATT, "var"));
+    public static final Unit<Power> KILOVAR = addUnit(MetricPrefix.KILO(VAR));
+    public static final Unit<Energy> VAR_HOUR = addUnit(new ProductUnit<>(VAR.multiply(Units.HOUR)),
             Energy.class);
+    public static final Unit<Energy> KILOVAR_HOUR = addUnit(MetricPrefix.KILO(VAR_HOUR));
+    public static final Unit<Power> VOLT_AMPERE = addUnit(new AlternateUnit<Power>(Units.WATT, "VA"));
+    public static final Unit<Power> KILOVOLT_AMPERE = addUnit(MetricPrefix.KILO(VOLT_AMPERE));
+    public static final Unit<Energy> VOLT_AMPERE_HOUR = addUnit(new ProductUnit<>(VOLT_AMPERE.multiply(Units.HOUR)),
+            Energy.class);
+    public static final Unit<Energy> KILOVOLT_AMPERE_HOUR = addUnit(MetricPrefix.KILO(VOLT_AMPERE_HOUR));
```

Comment:
see above

Reply:
I will remove the kilo, I'm absolutely fine with that.

Reply:
removed.